### PR TITLE
Create Jonnie-Dev.json

### PR DESCRIPTION
**Pull request by Jonnie-Dev**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4707"></a>

### CODE REVIEW

File: data/Jonnie-Dev.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "John Areola",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software engineer - 👨‍💻 Front end developer || UI Designer",
+  "avatar": "https://github.com/Jonnie-Dev.png",
```
Comment (stevenovak123) [inline data/Jonnie-Dev.json:6]: `avatar` is not a required property. GH actions auto pick it from your profile. Linking you a complete JSON file https://linkfree.eddiehub.io/docs/full-profile-example

File: data/Jonnie-Dev.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "John Areola",
+  "displayStatsPublic": true,
```
Comment (stevenovak123) [inline data/Jonnie-Dev.json:3]: This is now not required!

File: data/Jonnie-Dev.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  "type": "personal",
+  "bio": "Software engineer - 👨‍💻 Front end developer || UI Designer",
+  "avatar": "https://github.com/Jonnie-Dev.png",
+  "tags": ["frontend", "Open Source", "design", "internship"],
+  "socials": [
+    {
+      "platform": "twitter",
```
Comment (stevenovak123) [inline data/Jonnie-Dev.json:10]: `socials` follow a new system now.  pasting an example here but please refer to the link mentioned above 
`"socials": [
    { "icon": "FaTwitter", "url": "https://twitter.com/eddiejaoude" },
    { "icon": "FaGithub", "url": "https://github.com/eddiejaoude" },
    { "icon": "FaYoutube", "url": "https://youtube.com/eddiejaoude" }
  ],`

File: data/Jonnie-Dev.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      "url": "https://twitter.com/JonnieDev"
+    }
+  ],
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/JonnieDev",
+      "icon": "twitter"
```
Comment (stevenovak123) [inline data/Jonnie-Dev.json:18]: `icon` follows as `"icon": "FaTwitter"`

File: data/Jonnie-Dev.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/john-areola",
+      "icon": "linkedin"
```
Comment (stevenovak123) [inline data/Jonnie-Dev.json:23]: `icon` follows as `"icon": "FaLinkedin"`

File: data/Jonnie-Dev.json [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/Jonnie-Dev",
+      "icon": "github"
```
Comment (stevenovak123) [inline data/Jonnie-Dev.json:28]: `icon` follows as `"icon": "FaGithub"`

**Review by stevenovak123 — CHANGES_REQUESTED** [review]
Please refer to https://linkfree.eddiehub.io/docs/full-profile-example and correct the changes. Thanks

**Comment by Jonnie-Dev** [comment]
all suggestions taken and corrected

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**